packages/next/build/compiler.ts

```diff
@@ -40,10 +40,6 @@ export function runCompiler(
       (err: Error, statsOrMultiStats: { stats: Stats[] } | Stats) => {
         closeCompiler(compiler).then(() => {
           if (err) {
-            const reason = err?.toString()
```

Comment:
This caused the error thrown by webpack to be super obfuscated. It generally only happens when webpack (or plugins in certain cases) throw an error, so it's more helpful to show the full stacktrace